Gems/Atom/RHI/Metal/Code/Source/RHI/Scope.cpp

```diff
@@ -96,7 +96,7 @@ namespace AZ
             
             for (RHI::ImageScopeAttachment* scopeAttachment : GetImageAttachments())
             {
-                m_isWritingToSwapChainScope = 
+                bool isWritingToSwapChainScope = 
```

Comment:
I don't think you can do this. m_isWritingToSwapChainScope is used in other methods